terraform/modules/canary/data/helm-release.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
```

Comment:
I think this should all be generated using the "flux" module the same way we do for all the other flux releases ie:

https://github.com/alphagov/gsp-teams/blob/master/terraform/clusters/cluster.re-managed-observe-staging.aws.ext.govsvc.uk/cluster.tf#L58-#L84